File: Jt76.Ui/ClientApp/src/app/modules/admin/models/log-file.ts
export interface ILogFile {
  fileName: string;
  fileLocation: string;
  applicationName: string;
  recentFileLines: Array<String>;
  fullFile: Array<String>;
}


export class LogFile implements ILogFile {
  fileName = "";
  fileLocation = "";
  applicationName = "";
  recentFileLines = new Array<String>();
  fullFile = new Array<String>();
}

File: Jt76.Common/Services/FileService/LogFile.cs
namespace Jt76.Common.Services.FileService
{
	public class LogFile
	{
		public string FileLocation
		{
			get;
			set;
		}

		public string FileName
		{
			get;
			set;
		}

		public string ApplicationName
		{
			get;
			set;
		}

		public string[] RecentFileLines
		{
			get;
			set;
		}

		public string[] FullFile
		{
			get;
			set;
		} = new string[0];
	}
}
